guidelines/sc/20/visual-presentation.html

```diff
@@ -13,5 +13,7 @@ <h4>Visual Presentation</h4>
      <li>Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.</li>   
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
+
+  <p class="note">The 'mechanism' is usually the user-agent or assistive technology manipulating the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing in which case users might not use some of these mechanisms.</p>
```

Comment:
This definition of 'mechanism' doesn't quite match that already present in the [definition in the Understanding](https://www.w3.org/WAI/WCAG22/Understanding/visual-presentation.html#dfn-mechanism). It is also not clear that defining the mechanism is the main thurst of the identified issues.

The last sentence comes close to clarifying that the specifics may not be applicable to certain languages.

Reply:
I thought it did match, the first note under the definition is "The mechanism may be explicitly provided in the content, or may be [relied upon](https://www.w3.org/WAI/WCAG22/Understanding/visual-presentation.html#dfn-relied-upon) to be provided by either the platform or by [user agents](https://www.w3.org/WAI/WCAG22/Understanding/visual-presentation.html#dfn-user-agent)"

It was added because (from the meeting with the i18n folks) they were assuming they needed to use the values in their authored code.

Reply:
I'm not concerned about AT being included, but about the relationship between the mechanism and the UA.

To use an epub example, reading systems provide various controls for the properties mentioned by the SC. I don't think of the reading system itself as the mechanism, though. I manipulate the controls its provided, and the RS re-renders the layout. The RS provides the mechanism.

That's why I'm thinking it might more in line with the definition to say:

> The 'mechanism' is usually **provided by** the user-agent or assistive technology manipulating the content on behalf of the user.

If each control is a 'mechanism', the last sentence about users not needing all the mechanisms makes more sense.